src/common.ts

```diff
@@ -1,12 +1,13 @@
 import { Transaction } from '@solana/web3.js';
 
 export interface DIPDeposit {
-  splToken: string;
-  premiumAsset: string;
+  splTokenMint: string;
+  premiumAssetName: string;
   expirationMs: number;
-  strike: number;
-  type: string;
-  qty: number;
+  strikeUsdcPerToken: number;
+  // TODO: Remove this since all options are calls, just different tokens involved.
```

Comment:
Maybe it's implied but change todo to make helper fct to infer call or put from base or quote. We'll still need to use 'call' & 'put' in blackscholes dependency